bridges/snowbridge/primitives/core/src/outbound.rs

```diff
@@ -139,6 +139,39 @@ mod v1 {
 			// Fee multiplier
 			multiplier: UD60x18,
 		},
+		/// Transfer ERC20 tokens
+		TransferToken {
```

Comment:
Should be consistent with the new naming in `contracts/Types.sol`

```suggestion
		TransferNativeToken {
```

Reply:
https://github.com/Snowfork/polkadot-sdk/pull/128/commits/5ae8b0bdc1720dc71d0d7f3ab23843b3e319b85c